### ISSUE TITLE
t/16-expn.t fails if Devel::AssertOS::OSFeatures::SupportsSiebel is installed

### ISSUE THREAD

**Issue by eserte** [DrHyde/perl-modules-Devel-CheckOS#17]
See subject. Without `Devel::AssertOS::OSFeatures::SupportsSiebel` the test output looks like this:
```
...
ok 15 - OSFeatures::POSIXShellRedirection family has an expn()
ok 16 - QNX family has an expn()
...
```
And the failing variant looks like this:
```
...
ok 15 - OSFeatures::POSIXShellRedirection family has an expn()
ok 16 - OSFeatures::SupportsSiebel family has an expn()
ok 17 - QNX family has an expn()
...
# Looks like you planned 22 tests but ran 23.
```

Maybe working without a `plan` and with `done_testing` is best here.

**Comment by DrHyde** [comment]
Thanks, should be fixed in 1.79, which is on its way to the CPAN now.

**Comment by eserte** [comment]
Confirmed, looks good now.
